mbin_nmdc.wdl

```diff
@@ -178,6 +182,13 @@ task mbin_nmdc {
             mkdir -p gtdbtk-output
             echo "Mbin Sdb Could not be created for ~{name}" > mbin.sdb
         fi
+
+        if [ -f eukcc_output/eukcc.csv.final ]; then
+            echo "eukcc.csv.final exists."
+        else
+            mkdir -p eukcc_output
+            echo "No EUKCC2 result for ~{name}" > eukcc_output/eukcc.csv.final
```

Comment:
Checking to see if we should write empty results to a different file or to the same file.

Reply:
OK. This is to mimic what been implemented for other result above lines, including gtdbtk, checkm and the mbin.sdb.